requirements/dev.txt

```diff
@@ -220,7 +220,7 @@ pyproject-hooks==1.0.0
     #   -r requirements/pip-tools.txt
     #   build
     #   pip-tools
-pytest==8.1.0
+pytest==8.0.2
```

Comment:
version `8.1.0` was [yanked](https://pypi.org/project/pytest/8.1.0/) on PyPI due to some issues with plugins.